opacclient/opacapp/src/main/java/de/geeksfactory/opacclient/frontend/AccountFragment.java

```diff
@@ -1374,6 +1226,22 @@ public void onCollapseAnimationEnd() {
         refreshage();
     }
 
+    private void setHtmlTextOrHide(String value, TextView tv) {
+        if (value != null) {
```

Comment:
Is comparison with null good enough here or should we also check for empty strings?